src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,140 @@
+const MetaTrader          = require('../../../user/metatrader/metatrader');
+const getCurrencyFullName = require('../../../../common/currency').getCurrencyFullName;
+const Client              = require('../../../../base/client');
+const BinarySocket        = require('../../../../base/socket');
+const localize            = require('../../../../../_common/localize').localize;
+const Url                 = require('../../../../../_common/url');
+const isCryptocurrency    = require('../../../../../_common/base/currency_base').isCryptocurrency;
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $form,
+        $txt_other_reason,
+        $closure_loading,
+        $closure_container,
+        $success_msg,
+        $error_msg,
+        $crypto,
+        $fiat;
+
+    const onLoad = () => {
+        $txt_other_reason  = $('#other_reason');
+        $closure_loading   = $('#closure_loading');
+        $closure_container = $('#closure_container');
+        $success_msg       = $('#msg_main');
+        $error_msg         = $('#msg_form');
+        $crypto            = $('#crypto');
+        $fiat              = $('#change-fiat');
+        $form              = $(form_selector);
+
+        BinarySocket.wait('landing_company').then(() => {
+            const currencies = Client.currentLandingCompany().legal_allowed_currencies;
+            const current_currency = Client.get('currency');
+            const show_currencies = currencies.filter((val) => val !== current_currency);
```

Comment:
better also filter out other currencies that the client has since we have a full list of that, but up to UI/UX. you can look at the function `getCurrencies`